.github/workflows/macos.yml

```diff
@@ -14,7 +14,6 @@ jobs:
     runs-on: macos-latest
     if: github.event_name != 'pull_request' || contains('dreamer,kcgen,ant-222,Wengier', github.actor) == false
     strategy:
-      max-parallel: 3
```

Comment:
> When we go beyond the limit, GitHub CI simply does not guarantee that job will complete (it might be e.g. randomly cancelled).

The documentation says jobs are queued:

![2021-03-09_07-34](https://user-images.githubusercontent.com/1557255/110496072-fbf68e80-80a9-11eb-89bf-a2ac02c6be96.png)

https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration

---

One of the things I noticed is that some macOS virtual machines get disproportionately starved of CPU cycles, to the point where the starved VM will take 10-fold more time to do the same tasks as a non-starved.

I thought this could simply be one of our three macOS simply getting the short-end of the stick, in some unbalanced scheduling algorithm. 

So I tested `max_parallel` 3, 2, 1, and then serialized the release job after the debug builds. But even running 100% serialized didn't solve it: roughly every second VM _still_ was getting starved.  So it turn out our overall macOS times suffered, and the best overall throughput is best when we just run at full parallel.